src/atoms/Tab/Tab.native.js

```diff
@@ -0,0 +1,137 @@
+import React, { useState, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { View, TouchableOpacity } from 'react-native';
+import styled from 'styled-components/native';
+import Flex from '../Flex';
+import Text from '../Text';
+import { getColor, getIconKeys, makePxValue } from '../../_helpers/theme';
+import Icon from '../Icon';
+import Space from '../Space';
+import automation from '../../_helpers/automation-attributes';
+import isEmpty from '../../_helpers/isEmpty';
+
+const styles = {
+  label: {
+    borderBottomWidth({ isActive }) {
+      if (isActive) {
+        return '1px';
+      }
+      return '0px';
+    },
+    borderBottomColor({ isActive, theme, disabled }) {
+      if (disabled) {
+        if (isActive) {
+          return getColor(theme, 'primary.930');
+        }
+      }
+      if (isActive) {
+        return getColor(theme, 'primary.800');
+      }
+      return 'transparent';
+    },
+    margin() {
+      return [0, 0, 0, 0.5];
+    },
+    padding() {
+      return [1, 0, 1, 0];
+    },
+    color({ isActive, disabled, isPressed }) {
+      if (isPressed) {
+        if (isActive) {
+          return 'primary.900';
+        }
+        return 'shade.970';
+      } else if (isActive) {
+        return disabled ? 'primary.700' : 'primary.800';
+      } else {
+        return disabled ? 'shade.940' : 'shade.960';
+      }
+    },
+  },
+  tab: {
+    backgroundColor({ theme, isActive, isPressed }) {
+      if (isPressed) {
+        if (isActive) {
+          return getColor(theme, 'primary.920');
+        }
+        return getColor(theme, 'tone.940');
+      }
+      return 'transparent';
+    },
+  },
+};
+
+const StyledTabButton = styled(TouchableOpacity)`
+  min-width: ${makePxValue(6)};
+  background-color: ${styles.tab.backgroundColor};
+  border-top-left-radius: 2px;
+  border-top-right-radius: 2px;
+`;
+
+const Label = styled(View)`
+  border-bottom-width: ${styles.label.borderBottomWidth};
+  border-bottom-color: ${styles.label.borderBottomColor};
+`;
+
+const Tab = ({ label, isActive, icon, onPress, disabled, testID }) => {
+  const [isPressed, setTabPressed] = useState(false);
```

Comment:
`const [pressed, setPressed] = useState(false)`, since it is used in context of Tab component, we can avoid explicitly telling **tab**Pressed? 

And keeping consistency across states that define visuals of component e.g `pressed`, `disabled`, `active`.